### PR TITLE
alphabetized imports, make user id's constants

### PR DESCRIPTION
**Pull request by hvilander**

### CODE REVIEW

File: python_bot.py [hunk 1]
```diff
@@ -558,7 +570,7 @@ def emojify(string):
   elif(message.content.startswith('!ping')):
     await ping_command(message)
   elif(message.content.startswith('!stopvoice')):
-    if(message.author.id == '159785058381725696' or message.author.id == '328175857707253760'):
+    if(message.author.id == id_branden or message.author.id == '328175857707253760'):
```
Comment (20BBrown14) [inline python_bot.py]: nit: If you like you can remove this unknown user id. It's someone's for the intern discord server, where this bot originated. Or from my EQ2 guild discord. Either way, whoever owns this ID won't be using it.

File: python_bot.py [hunk 2]
```diff
@@ -567,7 +579,7 @@ def emojify(string):
   elif(message.content.startswith('!pizza')):
     await client.send_message(message.channel, 'Pizza? Who\'s paying for this? Not me.')
   elif(message.content.startswith('!Mugglewump')):
-    if(message.author.id == '159785058381725696' or message.author.id == '83809782691004416'):
+    if(message.author.id == id_branden or message.author.id == '83809782691004416'):
```
Comment (20BBrown14) [inline python_bot.py]: nit: Same here for this id. Infact, you could just remove this entire `Mugglewump` command if you want. This is someone from an old discord server

File: python_bot.py [hunk 3]
```diff
@@ -599,11 +611,11 @@ def emojify(string):
   elif(message.content.lower() == 'lol'):
     await client.send_message(message.channel if message.channel.name else message.author, 'lo\nlo\nlol')
   elif(message.content.lower() == ('!messwithkevin')):
-    if (message.author.id == '159785058381725696'):
+    if (message.author.id == id_branden):
       mess_with_kevin = not mess_with_kevin
       await client.send_message(message.channel if message.channel.name else message.author, 'Mess with kevin = ' + str(mess_with_kevin))
   elif(message.content.startswith('!voice')):
-    if(message.author.id == '159785058381725696' or message.author.id == '328175857707253760' or message.author.id == '314840626820677643' or message.author.id == '209415024354000897'): #These are user IDs and the logic only allows the players with this user ID to use this command
+    if(message.author.id == id_branden or message.author.id == '328175857707253760' or message.author.id == '314840626820677643' or message.author.id == '209415024354000897'): #These are user IDs and the logic only allows the players with this user ID to use this command
```
Comment (20BBrown14) [inline python_bot.py]: nit: Same with these IDs

**Comment by 20BBrown14** [comment]
minor: Don't forget to update version please.

**Review by 20BBrown14 — DISMISSED** [review]

File: python_bot.py (hunk 1 above)
Comment (20BBrown14) [reply python_bot.py]: Did you miss this one?

**Comment by 20BBrown14** [comment]
Looks like you missed the version too

**Review by 20BBrown14 — DISMISSED** [review]

File: python_bot.py (hunk 1 above)
Comment (hvilander) [reply python_bot.py]: yep, I was half awake when I was fiddling around this morning.

**Comment by hvilander** [comment]
last user id that I missed is fixed and I updated the version number.

**Review by 20BBrown14 — APPROVED** [review]

**Pull request merged**